saged/predict_tissue.py

```diff
@@ -428,7 +430,7 @@ def prep_tcga_data(args: argparse.Namespace) -> Tuple[datasets.RefineBioMixedDat
                 if args.weighted_loss:
                     loss_weights = utils.calculate_loss_weights(train_data)
                     supervised_config['loss_weights'] = loss_weights
-                if 'save_path' in supervised_config:
+                if 'save_path' in supervised_config and i == 0 and args.seed == 0:
```

Comment:
Might be good to add a comment explaining what you're doing here - looks like you only want to save the model info for the first iteration?